src/lib/theme/tokens.css

```
@@ -0,0 +1,121 @@
+
+/*
+ * Type and spacing.
+ *
+ * The interface is small and dense the way an editor's is; the manuscript is
+ * not, because it has to look like the page it will be printed as. Both are
+ * user-adjustable, since a 14px UI is unusable for some people and the app is
+ * meant to be lived in.
+ */
+:root {
+	--ui-font:
+		ui-sans-serif, system-ui, -apple-system, 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)